suite-native/storage/src/storage.ts

```diff
@@ -16,18 +16,32 @@ export const ENCRYPTED_STORAGE_ID = 'trezorSuite-app-storage';
 export let encryptedStorage: MMKV;
 
 const retrieveStorageEncryptionKey = async () => {
+    let secureKey: string | null = null;
     try {
-        const secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
-
-        if (secureKey) return secureKey;
+        secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
     } catch (error) {
-        // Some users are facing an error when they uninstall the app and then reinstall it,
-        // see https://github.com/expo/expo/issues/23426
-        await SecureStore.deleteItemAsync(ENCRYPTION_KEY);
-        captureException(error);
+        // If there is an error, report it and try to read one more time.
+        captureException(error, { tags: { attempt: 1 } });
+        try {
+            // There were some trouble reading from the SecureStore,
+            // let's wait a bit to make sure it wasn't just temporary error.
+            await new Promise(resolve => setTimeout(resolve, 100));
+            secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
+        } catch (error) {
+            captureException(error, { tags: { attempt: 2 } });
```

Comment:
How is second attempt triggered? I don't see any invokation of this function again

Reply:
The second attempt is triggered by the first attempt firing any error.